docs/CHANGELOG.asciidoc

```diff
@@ -162,10 +163,11 @@ Alerting users who are running 8.2 should not upgrade to either 8.3.0 or 8.3.1.
 Both 8.3.0 and 8.3.1 have a bug where alerting rules that were created or edited
 in 8.2 will stop running on upgrade. If you have upgraded to 8.3.0 or 8.3.1 and
 your alerting rules have stopped running with an error similar to the following
-example, you will need to use the *Update API key* operation in
-*{stack-manage-app} > {rules-ui}* to restore the alerting rule to a functional
-state. It should be noted that the *Update API Key* operation causes the
-alerting rule to run as the user that performed the operation. For more details
+example, you will need to go to *Stack Management* > *Rules and Connectors*, 
+multi-select the failed rules that need to be fixed, click on *Manage rules* > *Disable*, 
+then click on *Manage rules* > *Enable* to restore the alerting rule to a functional
+state. It should be noted that the disable-enable operation causes the 
+alerting rule to run as the user that performed the operation and reset the rule state. For more details
 about API key authorization, refer to <<alerting-authorization>>.
 
 Example error message::
```

Comment:
Add the following after the error message:

To fix the failed rules:

. Open then main menu, then click **Stack Management** > **Rules and Connectors**.

. Select the failed rules that you want to fix.

. Click **Manage rules** > **Disable**. 

. To restore the rule, click **Manage rules** > **Enable**.

NOTE: The rule runs as the user that disables or enables the rule and resets the rule state. For more information about API key authorization, check <<alerting-authorization>>.